go.mod

```diff
@@ -0,0 +1,107 @@
+	github.com/docker/go-connections v0.4.0 // indirect
+	github.com/docker/go-metrics v0.0.1 // indirect
+	github.com/docker/go-units v0.4.0 // indirect
+	github.com/docker/libtrust v0.0.0-20160708172513-aabc10ec26b7 // indirect
+	github.com/evanphx/json-patch v4.5.0+incompatible // indirect
+	github.com/ghodss/yaml v1.0.0
+	github.com/go-log/log v0.1.0 // indirect
+	github.com/godbus/dbus v0.0.0-20181101234600-2ff6f7ffd60f // indirect
+	github.com/gogo/protobuf v1.2.2-0.20190723190241-65acae22fc9d // indirect
+	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
+	github.com/golang/groupcache v0.0.0-20190702054246-869f871628b6 // indirect
+	github.com/google/renameio v0.1.0
+	github.com/google/uuid v1.1.1 // indirect
+	github.com/googleapis/gnostic v0.3.0 // indirect
+	github.com/gorilla/mux v1.7.3 // indirect
+	github.com/hashicorp/golang-lru v0.5.1 // indirect
+	github.com/huandu/xstrings v1.2.0 // indirect
+	github.com/imdario/mergo v0.3.7
+	github.com/joho/godotenv v1.3.0
+	github.com/konsorten/go-windows-terminal-sequences v1.0.2 // indirect
+	github.com/kubernetes-sigs/cri-o v1.12.10
+	github.com/modern-go/reflect2 v1.0.1 // indirect
+	github.com/mtrmac/gpgme v0.0.0-20170102180018-b2432428689c // indirect
+	github.com/opencontainers/go-digest v1.0.0-rc1
+	github.com/opencontainers/image-spec v1.0.1 // indirect
+	github.com/opencontainers/runc v0.1.1 // indirect
+	github.com/opencontainers/runtime-spec v0.0.0-20190207185410-29686dbc5559 // indirect
+	github.com/openshift/api v0.0.0-20190725193935-b7d4eb0fa1e0
+	github.com/openshift/client-go v0.0.0-20190627172412-c44a8b61b9f4
+	github.com/openshift/cluster-api v0.0.0-20190829140302-072f7d777dc8
+	github.com/openshift/library-go v0.0.0-20190704075327-f8abdcd57c46
+	github.com/pborman/uuid v0.0.0-20180906182336-adf5a7427709 // indirect
+	github.com/pkg/errors v0.8.1
+	github.com/sirupsen/logrus v1.4.2 // indirect
+	github.com/spf13/cobra v0.0.5
+	github.com/spf13/pflag v1.0.3
+	github.com/stretchr/testify v1.3.0
+	github.com/vincent-petithory/dataurl v0.0.0-20160330182126-9a301d65acbb
+	github.com/xeipuuv/gojsonpointer v0.0.0-20190905194746-02993c407bfb // indirect
+	github.com/xeipuuv/gojsonreference v0.0.0-20180127040603-bd5ef7bd5415 // indirect
+	github.com/xeipuuv/gojsonschema v1.1.0 // indirect
+	go4.org v0.0.0-20190313082347-94abd6928b1d // indirect
+	golang.org/x/crypto v0.0.0-20190701094942-4def268fd1a4 // indirect
+	golang.org/x/net v0.0.0-20190812203447-cdfb69ac37fc // indirect
+	golang.org/x/oauth2 v0.0.0-20190604053449-0f29369cfe45 // indirect
+	golang.org/x/time v0.0.0-20190308202827-9d24e82272b4
+	golang.org/x/tools v0.0.0-20190711191110-9a621aea19f8 // indirect
+	google.golang.org/appengine v1.6.1 // indirect
+	google.golang.org/genproto v0.0.0-20190708153700-3bdd9d9f5532 // indirect
+	google.golang.org/grpc v1.22.0 // indirect
+	gopkg.in/inf.v0 v0.9.1 // indirect
+	gotest.tools v2.2.0+incompatible // indirect
+	k8s.io/api v0.0.0-00010101000000-000000000000
```

Comment:
why don't we have clean tags like go dep?

Reply:
version for this is in the replace directive

Reply:
I think any of the weird `v0.0.0-00010101000000-000000000000` version tags can be shortened to just `v0.0.0` for readability since we are replacing the dependency down below, although that's definitely not required (see https://github.com/kubernetes/minikube/blob/master/go.mod#L79 for an example).

Reply:
I'll do that if I need to push this branch again for whatever reason